Makefile

```diff
@@ -40,7 +40,19 @@ from-scratch: ## start development environment from scratch
 	make up
 
 devstack-install: ## install local version of package into docker devstack for development
-	docker exec -t edx.devstack.lms bash -c 'source /edx/app/edxapp/nodeenvs/edxapp/bin/activate && cd /edx/app/edxapp/edx-platform && npm uninstall @edx/studio-frontend && cd /edx/src/studio-frontend && npm link && cd /edx/app/edxapp/edx-platform && npm link @edx/studio-frontend'
+	# step 0: remove studio-frontend from edx-platform/node_modules
+	docker exec -t edx.devstack.lms bash -c 'source /edx/app/edxapp/nodeenvs/edxapp/bin/activate && cd /edx/app/edxapp/edx-platform && npm uninstall @edx/studio-frontend && rm -rf ./node_modules/@edx/studio-frontend'
+	# step 1: create symlink in /edx/src/studio-frontend/symlinks
+	# note that we delete studio-frontend/node_modules; this is deliberate
+	docker exec -t edx.devstack.lms bash -c 'mkdir -p /edx/src/studio-frontend/symlinks && rm -rf /edx/src/studio-frontend/symlinks/* && source /edx/app/edxapp/edxapp_env && cd /edx/src/studio-frontend && export npm_config_prefix=/edx/src/studio-frontend/symlinks && npm link && rm -rf /edx/src/studio-frontend/node_modules'
+	# step 2: install symlinked package under edx-platform
+	# if studio-frontend/node_modules exists during this step, it can cause issues
+	docker exec -t edx.devstack.lms bash -c 'source /edx/app/edxapp/edxapp_env && cd /edx/app/edxapp/edx-platform && export npm_config_prefix=/edx/src/studio-frontend/symlinks && npm link @edx/studio-frontend'
+	# step 3: repair installations
+	docker exec -t edx.devstack.lms bash -c 'source /edx/app/edxapp/edxapp_env && cd /edx/app/edxapp/edx-platform && npm update && npm install'
+	docker exec -t edx.devstack.lms bash -c 'source /edx/app/edxapp/edxapp_env && cd /edx/app/edxapp/edx-platform/node_modules/@edx/studio-frontend && npm install && npm run build'
+	# step 4: update assets properly, so webservers know about them
+	docker exec -t edx.devstack.lms bash -c 'paver update_assets'
```

Comment:
Finally, `update_assets`. This will have the effect of:

* running webpack
* collecting generated assets and placing them in a place where the CMS may find them